pyogrio/_err.pyx

```diff
@@ -248,3 +344,197 @@ cdef void error_handler(CPLErr err_class, int err_no, const char* err_msg) nogil
 
 def _register_error_handler():
     CPLPushErrorHandler(<CPLErrorHandler>error_handler)
+
+cpl_errs = GDALErrCtxManager()
+
+
+cdef class StackChecker:
+
+    def __init__(self, error_stack=None):
+        self.error_stack = error_stack or {}
+
+    cdef int exc_wrap_int(self, int err) except -1:
```

Comment:
```suggestion
    cdef int check_int(self, int err) except -1:
```

I wonder if it is time for us to diverge from Fiona's original naming here and try to make this a bit more clear.
(also good to consider doing this for the top-level function names too.)

Reply:
ok, renamed